src/handler/guild/channel.ts

```diff
@@ -82,53 +80,15 @@ export class Channel {
         const channel = interaction.options.getChannel("channel", true);
         const type = interaction.options.getString("type", true);
 
-        if (type === "command") {
-            await this.commandChannelAdd(interaction, channel.id);
-        } else {
-            await this.uniqueChannelAdd(interaction, channel.id, type);
-        }
-    }
-
-    async commandChannelAdd(
```

Comment:
i thought we discussed keeping command channel? so it can easily be auto setup without having to mess with settings?

Reply:
We did? Even in that case it would only make sense once we have access to the new permission api though and code removed here would still be obsolete